var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -525,10 +543,17 @@ def determine_bootstrap_options(self, spec, with_libs, options):
                 # wrappers.  Since Boost doesn't use the MPI C++ bindings,
                 # that can be used as a compiler option instead.
                 mpi_line = "using mpi : %s" % spec["mpi"].mpicxx
+                if "platform=cray" in spec:
+                    mpi_line += " : <define>MPICH_SKIP_MPICXX"
```

Comment:
(question) have you tested this on Cray? Most systems that are Crays should actually report Linux as the platform (e.g. `spack arch`).

Reply:
This was the result of a bad conflict resolution. should be fixed now.